www/docs/nextjs/ssr.md

```diff
@@ -131,29 +131,27 @@ export async function getServerSideProps(
     ctx: {},
     transformer: superjson, // optional - adds superjson serialization
   });
-  const id = context.params?.id as string;
+
+  const id = String(context.params?.id);
```

Comment:
cc @armancharan this is all you need - using zod is a bit over-engineering. Casting is just fine since the API will yell at you if it's wrong